subprojects/smoke-test/smoke-test.gradle.kts

```diff
@@ -93,3 +95,26 @@ plugins.withType<EclipsePlugin>().configureEach { // lazy as plugin not applied
         plusConfigurations.add(smokeTestRuntimeClasspath)
     }
 }
+
+// TODO Copied from instant-execution.gradle.kts, we should have one place to clone this thing and clone it from there locally when needed
+tasks {
+    val santaTracker by registering(RemoteProject::class) {
+        remoteUri.set("https://github.com/gradle/santa-tracker-android.git")
+        ref.set("036aad22af993d2f564a6a15d6a7b9706ba37d8e")
```

Comment:
Can you comment here what branch this is from?

Reply:
Sure.